sourcemash/frontend/static/js/views/item_card_view.js

```diff
@@ -70,16 +93,20 @@ Sourcemash.Views.ItemCardView = Backbone.View.extend({
             this.model.save({saved: false},
                 {success: this.savedToast});
         } else {
-            this.model.save({saved: true},
-                {success: this.savedToast});
-
-            mixpanel.track("Saved", { "Item Title": this.model.get('title'),
-                                    "Feed Title": this.model.feed.get('title') })
-
-            if (!this.model.feed.get('subscribed')) {
-                this.showSubscribeModal({'source':'saved'});
-            }
-        }
+            if (!this.user.get('id')) {
```

Comment:
Not sure if I love the idea of keeping the view at the "higher" level and having clicks on each smaller view be passed "up" and captured by that global. It could definitely work, but the composition/structure seems non-ideal.